src/xcode/ENA/ENA/Source/Scenes/Checkin/QRCodeScanner/CheckinQRCodeScannerViewController.swift

```diff
@@ -42,17 +58,16 @@ class CheckinQRCodeScannerViewController: UIViewController {
 
 	override func viewWillDisappear(_ animated: Bool) {
 		super.viewWillDisappear(animated)
-		viewModel.deactivateScanning()
+		viewModel?.deactivateScanning()
```

Comment:
Why is the viewmodel optional?

Reply:
So we can access properties within the closure 
<img width="904" alt="Screenshot 2021-04-06 at 10 27 33" src="https://user-images.githubusercontent.com/15270737/113681805-b6d27600-96c2-11eb-8334-bf1592e94493.png">

<img width="924" alt="Screenshot 2021-04-06 at 10 26 08" src="https://user-images.githubusercontent.com/15270737/113681661-838fe700-96c2-11eb-9fa9-af9c62c5a014.png">

Reply:
I understand. I would prefer to have the 2 callbacks as optional parameters of the view model. 
But this ways works too, just a little confusing.